### DISCUSSION TITLE
SwiftUI

### DISCUSSION THREAD

**Discussion by bspringston** [WalletConnect/walletconnect-monorepo#1035] [Developer Support]
Is there an example for using WalletConnect v2 with SwiftUI?

**Comment by llbartekll** [comment]
We've added some SwiftUI components to example apps - still on `develop` branch.
We will be adding more in the future.

**Reply by Eunno-An** [reply]
can I get the branch link?

**Reply by chadyj** [reply]
Here you go @Eunno-An https://github.com/WalletConnect/WalletConnectSwiftV2/tree/develop

**Reply by Eunno-An** [reply]
Thank you! Is Example File = SwiftUI File?

**Reply by llbartekll** [reply]
Showcase app is built with SwiftUI

**Reply by Eunno-An** [reply]
How to build Showcase App?? @llbartekll @chadyj

**Reply by llbartekll** [reply]
by pressing Product->Build

**Reply by Eunno-An** [reply]
I have this error. hey @llbartekll Have met this problem?
I can't build. I set the build project to Showcase, But I can't build..
<img width="1073" alt="image" src="https://user-images.githubusercontent.com/33450365/201800409-306a1ab2-06d5-4fcc-8b5f-224ff22d6ba8.png">

**Comment by Eunno-An** [comment]
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/33450365/201305513-76b5513c-9702-44da-ad95-ac5fe47a446b.png">

I'm a newb, so I don't know well, but please understand this.
Please help me
Thanks
